package.json

```diff
@@ -8,13 +8,12 @@
     "node": ">=8.10.0"
   },
   "repository": "googleapis/nodejs-datastore",
-  "main": "./build/src/index.js",
+  "main": "build/src/index.js",
   "files": [
     "build/proto",
     "build/protos",
     "build/src"
   ],
-  "types": "build/src/index.d.ts",
```

Comment:
Are you sure we don't need that?

Reply:
all tests seem to pass without this, but I'm not sure if we can safely remove it. For my understanding, it's here because we need types for `index.js` and we moved it to `index.ts` so maybe this `index.d.ts` is no longer needed? @alexander-fenster for confirmation, same thing happened to the converting API `bigtable` Thank you!